docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Pulling an Image from a Private Registry
+---
+
+{% capture overview %}
+
+This page shows how to create a Pod that uses a Secret to pull an image from a
+private Docker registry or repository.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+* {% include task-tutorial-prereqs.md %}
+
+* To do this exercise, you need a
+[Docker ID](https://docs.docker.com/docker-id/) and password.
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Logging in to Docker
+
+    docker login
+
+When prompted, enter your Docker username and password.
+
+The login process creates or updates a `config.json` file that holds an
+authorization token.
+
+View the `configfile.json` file:
+
+    cat ~/.docker/config.json
+
+The output contains a section similar to this:
+
+    {
+        "auths": {
+            "https://index.docker.io/v1/": {
+                "auth": "c3RldmU1MzpTdGV2ZURvY2tAIzE2"
+            }
+        }
+    }
+
+### Creating a Secret that holds your authorization token
+
+Create a Secret named `regsecret`:
+
+    kubectl create secret docker-registry regsecret
```

Comment:
When I try to run this after logging into docker I get:

kubectl create secret docker-registry docker-reg
error: flag docker-username is required
See 'kubectl create secret docker-registry -h' for help and examples.